uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -104,6 +104,7 @@ public class MqttControlSteps {
 
     // publish default properties
     private static final boolean DEFAULT_PUBLISH_RETAIN = false;
+    private static final String DEFAULT_CONTENT_TYPE = null;
```

Comment:
Is that for publishing or expected to be received ?

Reply:
this is the default value for both of them.